Specifications/Language/2_Statements/Iterations.md

```diff
@@ -28,9 +28,9 @@ for (index, measured) in results {
 }
 ```
 
-## *Target-Specific Restrictions*
+## Target-specific restrictions
```

Comment:
@Bradben a minor quip -- this change removed the italics on this heading.

A similar heading exists at https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/2_Statements/ConditionalBranching.md#target-specific-restrictions which retains that style. Might be a good idea to make the two consistent.